dockers/docker-dhcp-relay/start.sh

```diff
@@ -24,3 +24,8 @@ if [ $(supervisorctl status | grep -c "^dhcp-relay:") -gt 0 ]; then
     # lifetime of the process.
     /usr/bin/wait_for_intf.sh
 fi
+
+j2 -f json --import-env=ENVIRONMENT /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2 /usr/share/sonic/templates/rsyslog_plugin/events_info.json  > /etc/rsyslog.d/dhcp_relay_events.conf
+rm -f /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2
+rm -f /usr/share/sonic/templates/rsyslog_plugin/events_info.json
```

Comment:
Are these 2 lines are really needed? Normally we do not remove rendering sources after rendering.

Reply:
Removed